chore(rn-storage): Update outdated React Native dependencies (#3485)
urql-graphql/urql: .changeset/strange-drinks-sip.md

---
'@urql/storage-rn': minor
---

> Bumped RN Netinfo and Async Storage to latest version and added Netinfo v11 to peer dependency array

Reviewer: ```suggestion
Bump peer-dependency of `@react-native-community/netinfo` to allow v11
```

> Bumped RN Netinfo and Async Storage to latest version and added Netinfo v11 to peer dependency array

Reviewer 2: ```suggestion
Add `@react-native-community/netinfo@^11.0.0` to the list of supported peer dependencies
```

Just a quick note here ✌️ These are user-facing, so from a user’s perspective the only relevant change is the newly supported peer dependency, since `devDependencies` are only for ourselves (i.e. this repo)

I just had a quick look at whether any of the major versions had an actionable change for us, but it looks fine 👍
